code/style.css

```diff
@@ -1,7 +1,358 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  max-width: 1500px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 20px;
+
+}
+
+
+body {
+  margin: 0;
+}
+
+* {
+  font-family: 'Nunito', sans-serif;
+}
+
+.navBar {
+  height: 52px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding-bottom: 10px;
+  position: fixed;
+  z-index: 3;
+  padding-top: 15px;
+  left: 0;
+  right: 0;
+  top: 0;
+  background-color: #f1faee;
+}
+
+.navBar a {
+  padding: 10px;
+  font-size: 20px;
+  color:#1d3557;
+}
+
+@media screen and (max-width: 400px){
+  .navBar a {
+    font-size: 18px;
+  }
+}
+.hero {
+  position: relative;
+  top: 0;
+  z-index: 1;
+  background: #a8dadc url('./assets/headerpicture.jpg') no-repeat center center;
+  background-size: cover;
+  margin-top: 52px;
+}
+
+
+.headerText {
+  position: absolute;
+  z-index: 2;
+  bottom: 0;
+  top: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  text-align: center;
+  background-color: rgba(0, 0, 0, 0.19);
+  color: #f1faee;
+  font-size: 25px;
+  font-family: 'Nunito', sans-serif;
+  
+}
+
+@media screen and (max-width: 400px) {
+  .headerText {
+    position: relative;
+    margin-top: 52px;
+  }
+}
```

Comment:
This media query is the same as the above one. You could just include this in the same :)